client/components/datapicker/datapicker.mjs

```diff
@@ -0,0 +1,168 @@
+const date = new Date();
+const calendarBox = document.querySelector(".dropdown");
+const dropdown = document.getElementById("myDropdown");
+const monthDays = document.querySelector(".days");
+const nonActiveInput = document.querySelector(".calendar-input-label")
```

Comment:
I already mentioned above problem with naming, but you can see problem even better here.
Your variable names are completely different from your classes. This means something went wrong in naming structure, you need to reconsider.

Reply:
changed